ipapython/certdb.py

```diff
@@ -290,7 +285,7 @@ def import_files(self, files, db_password_filename, import_keys=False,
                                     filename, line, e)
                             continue
                         else:
-                            extracted_certs += result.output + '\n'
+                            extracted_certs += '\n'.join(certs) + '\n'
```

Comment:
The line is ugly. Please convert extracted_certs to a list, append/extend the list and perform ```'\n\.join(extracted_certs)``` at the end of the function.